Formula/ruby@2.3.rb

```diff
@@ -37,10 +25,14 @@ class RubyAT23 < Formula
   depends_on "openssl"
   depends_on :x11 if build.with? "tcltk"
 
-  # https://www.ruby-lang.org/en/news/2017/08/29/multiple-vulnerabilities-in-rubygems/
-  patch :p0 do
-    url "https://bugs.ruby-lang.org/attachments/download/6691/rubygems-2613-ruby23.patch"
-    sha256 "510567a43d57ea9c8c7436b14e78d0a4d33380f410443dcf350b9867c9745748"
+  # Reverts an upstream commit which incorrectly tries to install headers
+  # into SDKROOT, if defined
+  # See https://bugs.ruby-lang.org/issues/11881
+  # The issue has been fixed on HEAD as of 1 Jan 2016, but has not been
+  # backported to the 2.3 branch yet and patch is still required.
+  patch do
```

Comment:
put it back lol

Reply:
But ILZ :sob:

Reply:
We'll need a component order audit for `patch do` before this becomes a reason to move things around. I figure eventually you'll break down and open a PR for that to `brew`.